dnf5/main.cpp

```diff
@@ -714,6 +724,15 @@ int main(int argc, char * argv[]) try {
 
             context.download_and_run(*context.get_transaction());
         }
+    } catch (libdnf::cli::GoalResolveError & ex) {
+        if (!any_repos_from_system_configuration && base.get_config().installroot().get_value() != "/" &&
+            !base.get_config().use_host_environment().get_value()) {
+            std::cout << "No repositories were loaded from the installroot. Pass --use-host-environment to use the "
+                         "configuration and repositories of the host system."
+                      << std::endl;
```

Comment:
Minor comment: 
```
Pass --use-host-environment to use the configuration and repositories of the host system.
```
What about to rotate the order. Firs - To use configuration and repositories of the host system pass <option>.
But please use your feelings more then my suggestion.

I somehow feel that the original statement is more strong then the second one - first condition then what to do.

Reply:
I agree, done